test/utils.js

```diff
@@ -36,11 +36,28 @@ export function almostEqual(a, b, criteria) {
   }
 }
 
-export function checkValue(
-    output, expected, criteria = opFp32AccuracyCriteria) {
+export function getBitwiseFloat32(value) {
```

Comment:
It probably makes sense to support other data types and make this function more generic, like `getBitwise`. It would also align with C++ implementation.